src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
```

Comment:
```suggestion
export async function getComments(postId: number) {
  try {
    const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
    if (!response.ok) {
      throw new Error(`${response.status} - ${response.statusText}`);
    }
    return response.json();
  } catch (error) {
    console.error(`Error fetching comments for postId ${postId}: ${error}`);
    throw error;
  }
}
```

it can help prevent errors. same in delComment and postComment